### DISCUSSION TITLE
新时代的阿Q罢了

### DISCUSSION THREAD

**Discussion by luowenfeng** [WeNeedHome/SummaryOfLoanSuspension#924] [General]
赵老爷家的狗腿子抢了阿Q的馒头.
阿Q求爷爷告奶奶也没能把馒头拿回来, 只好大声嚷嚷着让街坊邻居来评理.
"这狗腿子真不是个东西."
阿Q心里一惊, "别骂狗爷呀, 狗爷一个不乐意, 我的馒头就回不来了!"
"赵老爷也不是什么好货色"
阿Q大骇, "赵老爷一怒, 不但馒头讨不回来, 还得挨顿锤..."

公平?正义?民主?自由?
不能吃的阿Q不想要, 阿Q只想要回他的大白馒头.

太阳底下没有新鲜事...
时代变了, 舞台上的人只是换了身衣服.
作为看客, 能做的也只是哀其不幸罢了.
怒其不争? 大可不必. 阿Q不敢冲塔, 你我便敢了?
挂着梯子撺掇阿Q冲塔, 与吃人血馒头何异?

**Comment by Accelerator6666** [comment]
合着阿Q的东西被人抢了就算了？阿Q好不容易贷款买的馒头被人抢了，不仅馒头要不回来，贷款还得照样还？
阿Q心想反正馒头都没吃着，这贷款不还了还得挨你阿J的说教？

**Comment by zongjiangU** [comment]
冲塔就算了   但敢于为维护自己的利益发声，这是应该要做的。
阿Q不敢说话，是听坊间传闻上个讨要被埋了。
为何上个人被埋了，因为赵老爷害怕他出声音。
为何上个人可以被埋了，因为他是一个人，要是半个县城的人呢?
不要让  被剥削--恐惧--被剥削   形成闭环，形成恶心循环。
不主动如何破局，主动 不是做反人类反社会的事，而是为自己的利益发声，随着公信力的下降，局自然会有解。

**Comment by Siryedu** [comment]
其他非烂尾楼业主愿意帮他们发声，是因为大家有一个共同的目标“更公开透明的政府系统和更合理的权利制约和劳动 成果分配”，然后这棒烂尾楼业主里面有一部分只想着自己那个馒头，最后大家就内讧呗。

**Comment by clleo888** [comment]
当赵老爷要做永远主人的时候,阿Q非但没反对, 还表示全力支持
当赵老爷明明是个强盗而到处宣传自己如何伟大时,阿Q表示骗得好,反正自己是食饱了
当某些人被狗腿子骗,抢,姦时,阿Q认为与自己没关係,还认同他们维权是破坏社会和谐
当法律,自由任由赵老爷的人任意破坏时,阿Q表示做人就得服从,对反对而被打的人表示活该

结局大家都知道..当阿Q的馒头被抢后,没有任何法律,正义,说话的自由,一些外界想帮忙的人也已经没能力帮了,这全都是自己一点一点要来的,能怪谁?
